fast-list.js

```diff
@@ -283,6 +302,17 @@ FastList.prototype = {
 
       var section = source.getSectionFor(i);
       placeItem(item, i, section, geo, source, reload);
+
+      if (!source.populateItemDetail) {
+        return;
+      }
+
+      if (item.dataset.detailPopulated === 'true' || !geo.idle) {
+        return;
+      }
```

Comment:
I'm confused, if we abort `populateItemDetail()` here, does it get called later once the list has become `idle`?